### ISSUE TITLE
https://www.requestpolicy.com --> Expired Certificate

### ISSUE THREAD

**Issue by gserl** [RequestPolicyContinued/requestpolicy#711]
Just a heads up to whomever is in charge, that the SSL/TLS certificate for https://www.requestpolicy.com has expired:

www.requestpolicy.com uses an invalid security certificate. 
The certificate expired on 09/09/2015 01:22 PM. 
The current time is 09/23/2015 07:13 AM. (Error code: sec_error_expired_certificate)

**Comment by myrdd** [comment]
This is _not_ an issue of the RPC project.
Ping @jsamuel, what are you going to do with the website?

**Comment by genodeftest** [comment]
RPC contains links to https://www.requestpolicy.com/tutorial-1.0.html e.g. when using the installation assistant. Those links should be changed.

**Comment by DJ-Leith** [comment]
@myrdd, after the link to https://www.requestpolicy.com/tutorial-1.0.html is changed
please update this issue with the new link.

I'll update the 
**Updating from RP 0.5.xx to RPC**
(a linked topic at the end of the Quickstart [1] and [2])
which also has a link back to https://www.requestpolicy.com/tutorial-1.0.html

The linked topic text and screenshots are held in #665. 

[1] https://github.com/RequestPolicyContinued/requestpolicy/wiki/Quickstart

[2] https://requestpolicycontinued.github.io/Quickstart.html

DJ-Leith

**Comment by myrdd** [comment]
Isn't the "Quick Start" page (https://requestpolicycontinued.github.io/Quickstart.html) the new "tutorial" page?

**Comment by DJ-Leith** [comment]
I agree, it is the best page to link to.

I have updated the link in #665 to point to
https://requestpolicycontinued.github.io/Quickstart.html

However, last time I checked [1], the

"Teach me how to use RequestPolicy" button (shown next)


![fig-006-welcome-to-requestpolicy](https://cloud.githubusercontent.com/assets/12447809/7949641/d6a2cb02-0987-11e5-8d00-0fe7d00387b5.png)

**Fig-006-Welcome-to-RequestPolicy** (as seen in #665)
This button is also shown in Fig-008 and Fig-009.

The "Teach me ..." button is linked to the original website that Justin Samuel set up (which now has the expired certificate).

So, when you change the code behind the "Teach me ... " button please check the rest of the RPC codebase for other links to the 'old website'.  I don't know if there are any, I've not looked, but there might be a few.

[1]
I attempted to check again today.

I found it difficult to test.
**tl;dr** *Step 6* is the 'quick way to test'.

In the end, I cloned an existing Aurora Profile (which already had RPC 1.0.beta10.2)
which I am calling "T-check-RPC-Install".

**1.** Did an <code>"about:addons"</code>, "Check for Updates" to make sure all was up to date.
**2.** Closed this instance of Fx Dev Ed.
**3.** Directly edited the "prefs.js" in the "T-check-RPC-Install" Profile.

I set these two preferences 'back to false' (they were "true"), to simulate a fresh install of RPC.
user_pref("extensions.requestpolicy.initialSetupDialogShown", false);
user_pref("extensions.requestpolicy.welcomeWindowShown", false);

**4.** Started the "T-check-RPC-Install" Profile.

This showed: <code>about:requestpolicy?setup</code>

**5.** Confirmed that the "Teach me ..." button, in RPC 1.0.beta10.2, is still linking
to the 'old website'.

**6.** Using another Profile, without changing any settings, but going to <code>about:requestpolicy?setup</code>
also shows the "Teach me ..." button – the quick way to test.

DJ-Leith

**Comment by myrdd** [comment]
> I have updated the link in #665 to point to
> https://requestpolicycontinued.github.io/Quickstart.html

Good.

> please check the rest of the RPC codebase for other links to the 'old website'.

Yes, I'll do that. The links are

* https://www.requestpolicy.com/help-defaultpolicy.html,
* https://www.requestpolicy.com/tutorial-1.0.html and
* https://www.requestpolicy.com/help-rules.html

on the pages

* `about:requestpolicy?defaultpolicy`,
* `about:requestpolicy?setup` and
* `about:requestpolicy?yourpolicy`,

respectively.
